examples/tutorials/hello-gltf/vite.config.ts

```diff
@@ -0,0 +1,19 @@
+import {defineConfig} from 'vite';
+
+const alias = {
+  '@luma.gl/constants': `${__dirname}/../../../modules/constants/src`,
+  '@luma.gl/core': `${__dirname}/../../../modules/core/src`,
+  '@luma.gl/engine': `${__dirname}/../../../modules/engine/src`,
+  '@luma.gl/experimental': `${__dirname}/../../../modules/experimental/src`,
+  '@luma.gl/gltf': `${__dirname}/../../../modules/gltf/src`,
+  '@luma.gl/shadertools': `${__dirname}/../../../modules/shadertools/src`,
+  '@luma.gl/test-utils': `${__dirname}/../../../modules/test-utils/src`,
```

Comment:
We don't need the test-utils, but I guess for completeness sake... In some places (maybe loaders.gl examples) I replaced this explicit list with a loop that just aliases all subdirectories in `modules`.